### PR TITLE
fixed active menu option issue

### LINKED ISSUE #195: Fix Menu Option Active State Detection Issue
I have identified an issue where the menu option is not being recognized as active. Like whenever we go on a specific page, the user might get confused on which page he/she is on.

Screenshot 2023-10-11 at 9 48 51 PM

### PR DESCRIPTION
**Pull request by Huzaifa785**
## Fixes Issue

Fixed #195 

## Changes proposed

- [X] My code follows the code style of this project.
- [] My change requires changes to the documentation.
- [] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-10-11 at 9 57 03 PM

## Note to reviewers

I fixed the issue and now whichever page we go to, that nav option will be active respectively.

### CODE REVIEW

File: components/layout/sidebar-nav.tsx [hunk 1]
```diff
@@ -107,7 +107,7 @@ export function DocsSidebarNavItem({
         "bg-muted": pathname === item.href,
       })}
     >
-      {item.title}
+      {item.title}uh
```
Comment (Vishal-raj-1) [inline components/layout/sidebar-nav.tsx:110]: remove "uh"

File: components/code-editor.tsx [hunk 2]
```diff
@@ -134,7 +158,7 @@ export default function CodeEditor() {
                         />
                     </div>
                 </div>
-            )}
-        </>
-    );
+            )} */}
```
Comment (Vishal-raj-1) [inline components/code-editor.tsx:161]: why did you comment this section?

File: components/layout/sidebar-nav.tsx (hunk 1 above)
Comment (Huzaifa785) [reply components/layout/sidebar-nav.tsx:110]: will remove it soon

File: components/code-editor.tsx (hunk 2 above)
Comment (Huzaifa785) [reply components/code-editor.tsx:161]: sorry, I was testing something, I am making this file proper as well

**Comment by Huzaifa785** [comment]
I made the changes ✅

**Review by Vishal-raj-1 — APPROVED** [review]

**Pull request merged**